extensions-contrib/kafka-emitter/src/test/java/org/apache/druid/emitter/kafka/KafkaEmitterTest.java

```diff
@@ -106,7 +106,7 @@ public void testKafkaEmitter() throws InterruptedException
     ObjectMapper mapper = new ObjectMapper();
     mapper.registerModule(new JodaModule());
     final KafkaEmitter kafkaEmitter = new KafkaEmitter(
-        new KafkaEmitterConfig("", eventsType, "metrics", "alerts", requestTopic, "metadata", "test-cluster", null),
```

Comment:
Add a test to test the change from string to byte[]

Reply:
The change from string to byte[] is tested here: https://github.com/confluentinc/druid/pull/171/files#diff-640140e5e20f688c9d4b2f52c201a879d6c9d513f9ce16eb15f3e946f40ed306L114